src/components/FileRepo/DownloadButton.js

```diff
@@ -430,14 +434,7 @@ class DownloadButton extends React.Component {
 
   render() {
     const { isFileRepo = false } = this.props;
```

Comment:
return this.props.isFileRepo ? this.fileRepoRender() : this.cohortBuilderRender()